app/src/main/java/com/android/example/camerax/tflite/CameraActivity.kt

```diff
@@ -445,6 +445,8 @@ class CameraActivity : AppCompatActivity(), GLSurfaceView.Renderer {
             val approximateDistanceMeters = 2.0f
             // Performs a ray cast given a screen tap position.
             val results = frame.hitTestInstantPlacement(0F, 0F, approximateDistanceMeters)
+            val displayRotation = display!!.rotation
+            session.setDisplayGeometry(displayRotation, 256, 256)
```

Comment:
https://github.com/google-ar/arcore-android-sdk/issues/413

temporary hack for "Invalid ray produced by view data"